sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventDataForm.java

```diff
@@ -167,9 +166,9 @@ public class EventDataForm extends AbstractEditForm<EventDto> {
 	private final boolean inJurisdiction;
 	private List<UserReferenceDto> regionEventResponsibles = new ArrayList<>();
 	private List<UserReferenceDto> districtEventResponsibles = new ArrayList<>();
-	private EpidemiologicalEvidenceCheckBoxTree epidemiologicalEvidenceCheckBoxTree;
-	private LaboratoryDiagnosticEvidenceCheckBoxTree laboratoryDiagnosticEvidenceCheckBoxTree;
 	private LocationEditForm locationForm;
+	private CheckBoxTreeUpdated<EpidemiologicalEvidenceDetail> epidemiologicalEvidenceCheckBoxTreeUpdated;
```

Comment:
please convert these fields to local variable

Reply:
moved to local